include/mbedtls/ecdsa.h

```diff
@@ -294,6 +294,7 @@ int mbedtls_ecdsa_verify( mbedtls_ecp_group *grp,
  *                  size of the curve used, plus 9. For example, 73 Bytes if
  *                  a 256-bit curve is used. A buffer length of
  *                  #MBEDTLS_ECDSA_MAX_LEN is always safe.
+ * \param sig_size  The size of the \p sig buffer in bytes.
```

Comment:
It's not obvious to me whether
'size' = space available
'len' = length of actual data
is the clearest way to label these things. It seems like the other way round would fit equally well.
Perhaps something like `sig_space` or `sig_buf_len` would be clearer?

Reply:
(It occurs to me that there may be some convention of naming this way that I'm not aware of).

Reply:
Both the mbedtls_ API and the psa_ API consistently use “size” for the space available and “len(gth)” for the amount of data.